packages/core/src/components/account/MyAccountLayout/MyAccountLayout.tsx

```diff
@@ -0,0 +1,24 @@
+import type { PropsWithChildren } from 'react'
+import MyAccountMenu from '../MyAccountMenu'
+import menuRoutes from 'src/customizations/src/myAccount/navigation'
+import styles from '../section.module.scss'
+
+export type MyAccountLayoutProps = {}
+
+/* ######################################### */
+/* Mocked Data until development is finished */
+const username = 'John Cena'
```

Comment:
It will be the contract info. I talked to @ataideverton and we need to create the profile query with the contract info to get this information.